### DISCUSSION TITLE
For sale - 1 complete board and spare components as "partial kits"

### DISCUSSION THREAD

**Discussion by dasmith83** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#774] [General]
Hi all,

I made the decision to sell our Lay Z spa as I've lost interest in trying to maintain/fix our pump anymore lol. 

As such, I've got the below available/for sale:

1. Complete v2 board with red LLC (inc. nodemcu flashed with latest software version) - £25 inc. postage
2. 4 x partial kits inc. v2 board, red LLC, headers, 560 Ohm resistors, 6 pin JST male & female (no capacitors or nodeMCU) - £6 inc. postage

If anyone has any use for the above, let me know.

**Comment by Jan606** [comment]
Hello
I am interested, do you want to send it to the Netherlands, do you have a few photos, is the print the latest version? Thanks in advance Jan

**Comment by seesharpsteve** [comment]
I'm interested in a partial kit if you still have one
